cmd/plugin/builder/command/cli_compile.go

```diff
@@ -327,14 +330,17 @@ type target struct {
 	args []string
 }
 
-func (t target) build(targetPath, prefix, modPath, ldflags, tags string) error {
+func (t target) build(targetPath, prefix, modPath, ldflags, tags, goflags string) error {
 	cmd := goCommand("build")
 
 	var commonArgs = []string{
 		"-ldflags", ldflags,
 		"-tags", tags,
 	}
 
+	if goflags != "" {
+		cmd.Args = append(cmd.Args, goflags)
```

Comment:
I'm under the impression that as it is proposed, we would only be able to add a single goflag.  Say I want to add both `-buildvcs=false` and `-a`, I don't think I can.

Although we could allow for multiple `--goflags` flags, I think it would be simpler to accept multiple space-separated values for the single `--goflags` flag.
So, in the above line, you can probably just `strings.Split(goflags, " ")`

Then we should be able to do `tanzu builder plugin build --goflags "-buildvcs=false -a"`

Reply:
Great suggestion. I have applied it in the latest commit.